test/parallel/test-async-wrap-getasyncid.js

```diff
@@ -66,6 +66,12 @@ function testInitialized(req, ctor_name) {
 }
 
 
+{
+  // TODO: determine the rigth way to expose promise wrap.
+  new Promise((res) => res(5));
```

Comment:
I don't think we need to expose the `PromiseWrap` on the `new Promise` object. It just has to be the `resource` object in the init hook, and the `PromiseWrap` has to point to the `new Promise` object.

We don't expose `*Wrap` objects to userland in any other way than through the `async_hooks` `resource` object.